### ISSUE TITLE
WorkManager documentation for using Annotations

### ISSUE THREAD

**Issue by tyczj** [InsertKoinIO/koin#1487]
All the documentation for using androids work manager is for when using DSL, nothing shows how to setup work manager to be able to use with Annotations

**Comment by arnaudgiuliani** [comment]
Good catch. Can you reopen issue on https://github.com/InsertKoinIO/koin-annotations repo?

Thanks

**Comment by tyczj** [comment]
reopened over there
